### ISSUE TITLE
question about weight of reco loss

### ISSUE THREAD

**Issue by hxshust** [lorenmt/reco#14]
hi,thank you for your impressive work!
I have a question about weight of reco loss.In the paper, total_loss = reco_loss + sup_loss, but I found in my experiment  that the value of reco_loss is dozens of times that of sup_loss. If reco_loss is set with a small weight so that it and sup_loss reach an order of magnitude, will it be better?
Looking forward to your reply.

**Comment by lorenmt** [comment]
Hi,

Yes this makes sense. A small weighting on the reco loss in this case can be helpful. But still, you may need to do a hyper-paramter search to see the effect of performance with different task weightings.

Let me know if you have further questions.
